gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/InstrumentatedGoogleHadoopFileSystem.java

```diff
@@ -37,7 +38,12 @@
 
 public class InstrumentatedGoogleHadoopFileSystem extends GoogleHadoopFileSystemBase
     implements IOStatisticsSource {
-  GhfsInstrumentation instrumentation;
+
+  /** Instrumentation to track Statistics */
+  private GhfsInstrumentation instrumentation;
```

Comment:
IF this class is only responsible for IOStatistics, can we rename it accordingly ?

Reply:
We are using this class for also Storage Statistics